spm12/metadata/get_metadata_val_classic.m

```diff
@@ -804,7 +804,10 @@
                         parValue{cRes} = 230:-10:0;
                     elseif contains(valMODELNAME,'7T','IgnoreCase',true)
                         parLocation{cRes} = [nam{1} '.adFree(3:4)'];
-                        parValue{cRes} = val{1}.adFree(3):-val{1}.adFree(4):0;      
+                        parValue{cRes} = val{1}.adFree(3):-val{1}.adFree(4):0;
+                    elseif contains(valMODELNAME,'TrioTim','IgnoreCase',true)
+                        parLocation{cRes} = [];
+                        parValue{cRes} = [];
                     else
                         parLocation{cRes}=[];
                     end
```

Comment:
I think this would be a lot cleaner, and wouldn't require updating the code every time someone installs the sequence on a different scanner model:
```suggestion
                        parLocation{cRes}=[];
                        parValue{cRes} = [];
                    end
```